### ISSUE TITLE
Starter gives the incorrect remote link on newly started repo when using https link

### ISSUE THREAD

**Issue by ericjim** [rangle/rangle-starter#5]
I successfully start a new react redux project using rangle-starter.

```
ericjimenez@rngl ~/W/Rangle> rangle-starter react-redux-starter ericjim https://github.com/ericjim/react-playground.git
org: //github.com/ericjim folder: react-playground
Cloning into 'react-playground'...
Saving password to keychain failed
Identity added: /Users/ericjimenez/.ssh/id_rsa ((null))
remote: Counting objects: 87, done.
remote: Compressing objects: 100% (71/71), done.
remote: Total 87 (delta 5), reused 59 (delta 4), pack-reused 0
Receiving objects: 100% (87/87), 23.75 KiB | 0 bytes/s, done.
Resolving deltas: 100% (5/5), done.
Checking connectivity... done.
Initialized empty Git repository in /Users/ericjimenez/Workspace/Rangle/react-playground/.git/
[master (root-commit) 2be5f50] Initial commit from source:
 64 files changed, 1898 insertions(+)
 ....
```

However, my `origin` remote has an invalid link.

```
ericjimenez@rngl ~/W/R/react-playground> git push origin master
ssh: Could not resolve hostname https: nodename nor servname provided, or not known
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Upon further inspection, I see that my `origin` is missing the `//github.com`

```
ericjimenez@rngl ~/W/R/react-playground> git remote --verbose
origin  https:ericjim/react-playground.git (fetch)
origin  https:ericjim/react-playground.git (push)
upstream    https://github.com/ericjim/react-playground.git (fetch)
upstream    https://github.com/ericjim/react-playground.git (push)
```

With that said, since I am using two-step auth, I need to use an ssh key to push to my repo using the starter. So I change the remote to...

```
ericjimenez@rngl ~/W/R/react-playground> git remote rm origin
ericjimenez@rngl ~/W/R/react-playground> git remote add origin git@github.com:ericjim/react-playground.git
ericjimenez@rngl ~/W/R/react-playground> git push origin master
Counting objects: 87, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (75/75), done.
Writing objects: 100% (87/87), 23.54 KiB | 0 bytes/s, done.
Total 87 (delta 5), reused 0 (delta 0)
To git@github.com:ericjim/react-playground.git
 * [new branch]      master -> master
```

And that's how I circumvent the malformed origin link when using the starter.

**Comment by ericjim** [comment]
The problem is this line:

```
ericjimenez@rngl ~/W/Rangle> rangle-starter react-redux-starter ericjim https://github.com/ericjim/react-playground.git
```

I should've used the `git@github.com:ericjim/react-playground.git` to begin with.

**Comment by SethDavenport** [comment]
Yeah I tend to use ssh URLs.  If you feel inclined to make it work with HTTPS ones, go ahead and make a PR :)

**Comment by kosz** [comment]
Picking this up, should be easy at this point. 
I have some usability concerns first: 
- in order to implement this i'll have to increase the number of questions from 4 to 5. is the significant percentage increase worth the feature ? could it be handled differently ? 
- if we ask the question what should be the default ? if we make `https` default it may be friendlier to users who are not familiar with configuring ssh keys. if we make it `ssh` it would be more efficient for internal use.

**Comment by kosz** [comment]
@SethDavenport @ericjim would this work ? https://github.com/rangle/rangle-starter/pull/66

**Comment by JaKXz** [comment]
My vote is that SSH be default because it's (generally) faster and more secure - it's a best practice in and of itself.

**Comment by ericjim** [comment]
@kosz :tada:

**Comment by SethDavenport** [comment]
This fix has been published to npm in version 1.1.0.
